package.json

```diff
@@ -35,7 +35,9 @@
     "sense": "nebula sense --ext ./src/extension/ext-raw.js --meta ./src/extension/meta.json",
     "spec": "sy from-jsdoc -c ./spec-configs/props.conf.js",
     "start": "nebula serve --type sn-org-chart",
+    "start:mfe": "nebula serve --mfe --type sn-org-chart",
     "test:e2e": "playwright test",
+    "test:rendering": "playwright test --config=./playwright.config.js",
```

Comment:
Why add this? This should do the same as `test:e2e` (which is used in the CI pipeline). The config parameter is the default

Reply:
You are right, correction made in this new PR https://github.com/qlik-oss/sn-org-chart/pull/354 where I've split the code to include only property panel and property migration.